build.sbt

```diff
@@ -1,58 +1,57 @@
-organization        := "io.github.pauljamescleary"
-name                := "scala-pet-store"
-version             := "0.0.1-SNAPSHOT"
-scalaVersion        := "2.12.9"
-crossScalaVersions  := Seq("2.12.9", "2.13.0")
+organization := "io.github.pauljamescleary"
+name := "scala-pet-store"
+version := "0.0.1-SNAPSHOT"
+scalaVersion := "2.12.9"
+crossScalaVersions := Seq("2.12.9", "2.13.0")
```

Comment:
Current config is [`align=none`](https://scalameta.org/scalafmt/docs/configuration.html#align-none). I'm thinking we should consider using [`align=more`](https://scalameta.org/scalafmt/docs/configuration.html#align-most). Or we probably could use [`align.tokens`](https://scalameta.org/scalafmt/docs/configuration.html#aligntokens) to handle `=` and `:=` only.